parse/src/main/java/com/parse/ParsePlugins.java

```diff
@@ -178,7 +178,17 @@ InstallationId installationId() {
     File getCacheDir() {
         synchronized (lock) {
             if (cacheDir == null) {
-                cacheDir = new File(applicationContext.getCacheDir(), "com.parse");
+                /*
+                Check for old reference file before migration to v3.0.0.
+                If old reference found, copy the ref to new directory and delete the ref.
```

Comment:
I can see that you copy reference, but what's the idea behind @rommansabbir? Coping reference will no do the migration itself, right?

Where the actual data copy is done from the old to the new path aka. the migration?